common/Makefile.am

```diff
@@ -18,6 +18,12 @@ AM_CPPFLAGS = \
   -DXRDP_PID_PATH=\"${localstatedir}/run\" \
   -DXRDP_LOG_PATH=\"${localstatedir}/log\"
 
+if XRDP_DEBUG
+AM_CPPFLAGS += -DXRDP_DEBUG
+else
+AM_CPPFLAGS += -DXRDP_NODEBUG
```

Comment:
XRDP_NODEBUG is not used anywhere in the code. Instead of adding it, let's remove it from the makefiles where it's defined.